### PR TITLE
[MVC-1252] Uploaded Policy plan

### PR DESCRIPTION
**Pull request by nashthecoder**
**[MVC-1252] Uploaded Policy plan** 

- Extracted policy-plan, contact and policy code from _layout/simple.html into separate include files. 
- Added policy-plan.markdown
- Added _data/navigation.yml for footer-links. 
- Fixed privacy policy nav error on footer section.  Screenshot 2021-07-06 at 10 56 18

Uploaded Policy Plan Video
https://user-images.githubusercontent.com/15178823/124840531-de6ffd80-df93-11eb-94e5-ca1205eef424.mov

[MVC-1252]: https://triggerise.atlassian.net/browse/MVC-1252

### CODE REVIEW

File: _includes/policy-plan.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="wrapper policy-plan {% if page.title == "Policy Plan" %}{% endif %} uk-scrollspy-inview uk-animation-fade">
+    <p>{{ page.description | markdownify }}</p>
+</div>
+
+
+
```
Comment (raguiar9080) [inline _includes/policy-plan.html]: ✂️ empty lines

**Review by raguiar9080 — APPROVED** [review]
Looking real clean 👍

File: _includes/policy-plan.html (hunk 1 above)
Comment (nashthecoder) [reply _includes/policy-plan.html]: Opps...

Comment (nashthecoder) [reply _includes/policy-plan.html]: Cleaned up and merged.

**Pull request merged**